lib/ansible/modules/service.py

```diff
@@ -89,7 +89,12 @@
     - For AIX, group subsystem names can be used.
     - Supports C(check_mode).
 seealso:
-    - module: ansible.windows.win_service
+- module: ansible.builtin.sysvinit
+  description: Manage SysV services.
+- module: ansible.builtin.service
+  description: Generic module to manage a service.
```

Comment:
```suggestion
```
You probably want to reference `ansible.builtin.systemd` instead.

Reply:
indeed !